tasks/deploy.ts

```diff
@@ -2,19 +2,36 @@ import { HardhatEthersHelpers, HardhatRuntimeEnvironment } from 'hardhat/types';
 import fs from 'node:fs';
 import { ContractAddresses, NetworkConfig } from '../utils/scripts/types';
 import { parseJsonFile } from './utils';
+import {
+  deployRelayHub,
+  deployDefaultSmartWallet,
+  deployCustomSmartWallet,
+  deployNativeHolderSmartWallet,
+  deployUtilToken,
+  deployVersionRegistry,
+} from './deployers';
 
 const ADDRESS_FILE = process.env['ADDRESS_FILE'] || 'contract-addresses.json';
 
 export type AddressesConfig = { [key: string]: ContractAddresses };
 
+export type DeployArg = {
+  relayHub?: boolean;
+  defaultSmartWallet?: boolean;
+  customSmartWallet?: boolean;
+  nativeHolderSmartWallet?: boolean;
+  utilToken?: boolean;
+  versionRegistry?: boolean;
+};
+
 // TODO: Use the async version of fs.writeFile
 export const writeConfigToDisk = (config: NetworkConfig) => {
   fs.writeFileSync(ADDRESS_FILE, JSON.stringify(config));
   console.log(`Address file available at: "${ADDRESS_FILE}"`);
 };
 
 export const updateConfig = async (
```

Comment:
I tried to update the config after deploying multiple contracts, and if you deploy less contracts it does not erase the previous that are extra.